library/oraclelinux

```diff
@@ -1,13 +1,13 @@
 Maintainers: Oracle Linux Product Team <ol-ovm-info_ww@oracle.com> (@Oracle)
 GitRepo: https://github.com/oracle/container-images.git
-GitCommit: 01a15ec99c7470a3391c691509db1759b41eaf66
+GitCommit: ae7ed9f8a7fc53fcc06b7137748c3a3ef84db4d9
 GitFetch: refs/heads/master
 # https://github.com/oracle/container-images/tree/dist-amd64
 amd64-GitFetch: refs/heads/dist-amd64
 amd64-GitCommit: 6dffbbc3b9599111ca46443a92cee89d0e06ad82
```

Comment:
Probably wanted `ae7ed9f8a7fc53fcc06b7137748c3a3ef84db4d9` down here too? (or could just merge the amd64 values into the "global")

Reply:
Right. Fixed. :)